docs/book/products/search.rst

```diff
@@ -0,0 +1,55 @@
+.. index::
+   single: Search
+
+Search
+======
+
+Having a products search functionality in an ecommerce system is a very popular usecase. Sylius provides a products search functionality
+that is a grid filter.
+
+Grid filters
+------------
+
+For simple usecases of products search use the **filters of grids**.
+The products grid in the shop has already a ``search`` filter:
+
+.. code-blok:: yaml
+
+    # Sylius/Bundle/ShopBundle/Resources/config/grids/product.yml
+    filters:
+        search:
+            type: string
+            label: false
+            options:
+                fields: [translation.name]
+            form_options:
+                type: contains
+
+It searches by product names that contain a string that the user typed in.
```

Comment:
I still don't know how to use it. Do I need to send a request? What should it contain?